src/core/util/json/json_object_loader.h

```diff
@@ -388,6 +388,22 @@ class AutoLoader<std::map<std::string, T>> final : public LoadMap {
   ~AutoLoader() = default;
 };
 
+// Specializations of AutoLoader for maps with a custom comparator.
+template <typename T, typename C>
+class AutoLoader<std::map<std::string, T, C>> final : public LoadMap {
```

Comment:
I don't think we should need a new variant of this.  Instead, we should be able to just add the missing parameter to the existing variant for `std::map<>`.

Reply:
Changed to that